packages/contracts/contracts/tokens/GovernanceERC20.sol

```diff
@@ -18,40 +18,27 @@ contract GovernanceERC20 is AdaptiveERC165, ERC20VotesUpgradeable, Permissions {
 
      /// @notice The role identifier to mint new tokens
     bytes32 public constant TOKEN_MINTER_ROLE = keccak256("TOKEN_MINTER_ROLE");
-
-    /// @dev describes the version and contract for GSN compatibility.
-    function versionRecipient() external virtual override view returns (string memory) {
-        return "0.0.1+opengsn.recipient.GovernanceERC20";
-    }
-
-    function initialize(
+    
+    function __GovernanceERC20_init(
```

Comment:
what's the reasoning behind bringing `__GovernanceERC20_init` inside `initialize` ? just tried to put all the code that is inside `__GovernanceERC20_init` in the `initialize` function directly and it works. 

could you explain or have it the same way we had ?

Reply:
I wanted to maintain the possibility to inherit from the token, but I guess that's not needed, and we can save gas this way. I reverted the change.